scripts/add-extension.sh

```diff
@@ -16,7 +16,7 @@ if [[ -z "$EXTENSION_SETTINGS" ]]; then
     jq -n \
         --arg tag "$CHECKOUT_TAG" \
         --arg namespace "$RELEASE_NAMESPACE" \
-        '{properties: {extensionType: "Microsoft.openservicemesh", autoUpgradeMinorVersion: "false", version: $tag, releaseTrain: "Staging", scope: { cluster: { releaseNamespace: $namespace } } } }' > osm_extension.json
+        '{properties: {extensionType: "Microsoft.openservicemesh", autoUpgradeMinorVersion: "false", version: $tag, releaseTrain: "Staging", scope: { cluster: { releaseNamespace: $namespace } }, "configurationProtectedSettings": { "osm.OpenServiceMesh.fluentBit.logLevel": "debug", } } }' > osm_extension.json
```

Comment:
We are dropping this variable entirely in v0.8.0 and instead going to control log level using the controllerLogLevel variable defined [here](https://github.com/openservicemesh/osm/blob/main/charts/osm/values.yaml#L102). This change is going to be tested reflected in v0.8.0-rc.2 and will be updated in osm-azure by PR #71. Would it still make sense to merge this change?

Reply:
@SanyaKochhar Depends on when upstream will cut the change for controllerLogLevel. I think for now we can merge this and just collect debug level logs from osm-controller, and then you can override this in your PR when the change is reflected upstream.